platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/Migratable.java

```diff
@@ -37,15 +35,64 @@
  * </p>
  */
 public interface Migratable extends Describable {
+    /**
+     * Gets the current export version handled by this migratable.
+     * <p>
+     * When exporting data, the migration framework will export this version information and provide
+     * it back to the {@link #doImport} method as part of the import migration context when importing
+     * the corresponding exported data.
+     *
+     * @return the current version handled by this migratable
+     */
+    @Override
+    @Nullable
```

Comment:
❓ Why `@Nullable`? Shouldn't we require it? Any format information (e.g., `<Major>.<Minor>`) we should provide/enforce? Wonder if we should consider using this instead of a plain String: https://osgi.org/javadoc/r4v43/core/org/osgi/framework/Version.html or writing our own?